cli/smartcontract/smart_contract.go

```diff
@@ -679,7 +679,7 @@ func getAccFromContext(ctx *cli.Context) (*wallet.Account, error) {
 		return nil, cli.NewExitError(fmt.Errorf("wallet contains no account for '%s'", address.Uint160ToString(addr)), 1)
 	}
 
-	rawPass, err := input.ReadPassword(ctx.App.Writer,
+	rawPass, err := input.ReadPassword(ctx.App.ErrWriter,
```

Comment:
That's just not correct, stderr is for errors.

Reply:
Fixed.